packages/common/telemetry-constants.ts

```diff
@@ -1069,7 +1069,7 @@ export interface StudioPricingPlanCtaClickedEvent {
  */
 export interface StudioPricingSidePanelOpenedEvent {
   action: TelemetryActions.STUDIO_PRICING_SIDE_PANEL_OPENED
-  properties: { currentPlan?: string }
+  properties: { currentPlan?: string; source?: string }
```

Comment:
```suggestion
  properties: { 
    currentPlan?: string,
    
    /**
     * Tracks how user landed on the Pricing side panel, e.g. computeDiskManagement, restoreToNewProject
     */
    source?: string 
  }
```

Reply:
@SaxonF Ah started review before lunch and submitted after, so some of it is out of date now, but you get the gist!